lib/common/Memory/Chakra.Common.Memory.vcxproj

```diff
@@ -161,4 +191,4 @@
     <Import Project="$(VCTargetsPath)\BuildCustomizations\masm.targets" />
     <Import Project="$(BuildConfig_ARMASM_Path)armasm.targets" />
   </ImportGroup>
-</Project>
+</Project>
```

Comment:
linux doesn't care a whole lot about visual studio projects --- it's an IDE generated change. If preferred, I can edit it manually to re-add BOM and trailing line, but I doubt anyones tools are really impacted